capstone-rs/src/instruction.rs

```diff
@@ -178,21 +172,8 @@ macro_rules! impl_SliceIterator_wrapper {
     }
 }
 
-/// An iterator over the instructions returned by disasm
-///
-/// This is currently the only supported interface for reading them.
-pub struct InstructionIterator<'a>(slice::Iter<'a, cs_insn>);
-
-impl_SliceIterator_wrapper!(
```

Comment:
I think you can get rid of this macro definition as well (and all uses) if you make the same transparent/deref changes to the appropriate types.